.github/pull_request_template.md

```diff
@@ -7,35 +7,29 @@ that issue.
 
 ## Types of changes
 
-What types of changes does your code introduce to the project: _Put
-an `x` in the boxes that apply_
+What types of changes does your code introduce to the project?
+_Put an `x` in the boxes that apply_
 
 - [ ] Bugfix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing
-      functionality to not work as expected)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 
 ## Checklist
 
-_Put an `x` in the boxes that apply. You can also fill these out after
-creating the PR. If you're unsure about any of them, don't hesitate to
-ask. We're here to help! This is simply a reminder of what we are going
-to look for before merging your code._
+_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
 
 - [ ] I have read the [CONTRIBUTING][contrib] doc
 - [ ] I have signed the [CLA][cla]
-- [ ] I have added tests that prove my fix is effective or that my
-      feature works
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] I have run **all** the existing tests locally (not just those related to my feature) and there are no errors
+- [ ] After the last push to the PR branch, I have run the lint script locally and there are no changes to the code base
```

Comment:
i thought these lines were more than 80 chars, but markdown linter is not complaining.... so good to go

Reply:
mmm it maybe that being a "." directory is kind of hidden

Reply:
the problem is that the PR template renderer is broken. If you put this markdown anywhere in GH

```
This is a long sentence
I'd like to break
```

It gets rendered as

> This is a long sentence I'd like to break

except when it comes to the PR template where it'd be rendered as

> This is a long sentence
> I'd like to break

Reply:
That's the reason why I reformatted the file to keep paragraphs on one line, even if they go over 80 chars. It sucks and we can reformat again if the GH guys fix the renderer.